mtp_noms_ops/apps/security/forms/object_list.py

```diff
@@ -146,6 +158,29 @@ def get_query_data(self, allow_parameter_manipulation=True):
         return query_data
 
 
+class SendersFormV2(BaseSendersForm):
+    """
+    Search Form for Senders V2.
+    """
+    search = forms.CharField(
+        label=_('Search payment source name or email address'),
+        required=False,
+        help_text=_('Common or incomplete names may show many results'),
+    )
+
+    # NB: ensure that these templates are HTML-safe
+    filtered_description_template = 'Results containing {filter_description}, ordered by {ordering_description}.'
```

Comment:
we'll need to check this with the team. the content-reviewed prototypes do not mention ordering. our options are:
1) get a content review to ok this text
2) not mention ordering _at all_
3) have 2 descriptions! one short form for the visible text, the longer-form including ordering for the invisible table caption [probably my preferred option, though it adds complexity]

Reply:
We decided not to show the ordering at all for the time being.